boltrouter/bolt_request.go

```diff
@@ -47,127 +47,139 @@ var (
 // This new http.Request is routed to the correct Bolt endpoint and signed correctly.
```

Comment:
I think the comment in this method needs to be updated to also have gcp in it. I can't access the line in the github editor tho

Reply:
done.